packages/duppy-windows.0.7.0/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer: "Romain Beauxis <toots@rastageeks.org>"
+authors: "The Savonet Team <savonet-users@lists.sourceforge.net>"
+homepage: "https://github.com/savonet/ocaml-duppy"
+build: [
+  ["./configure" "--host=%{conf-gcc-windows:host}%" "--prefix=%{prefix}%" "OCAMLFIND_TOOLCHAIN=windows"]
+  [make]
+]
+install: [
+  ["env" "OCAMLFIND_TOOLCHAIN=windows"
+   "make" "install"]
+]
+remove: ["ocamlfind" "remove" "duppy"]
```

Comment:
Forgot -toolchain here. I think the CI needs to test removal as well.

Reply:
Good point. Will look into it.